fastlane/Fastfile

```diff
@@ -0,0 +1,20 @@
+default_platform(:ios)
+
+platform :ios do
+  desc "Build SwiftLeeds scheme"
+  lane :build do |options|
```

Comment:
is it worth putting a default for the configuration option?

Reply:
It's a good call, but we already have a separate lane for calling `build` with `Debug` option. 
I think it's good basic setup for clarity for now as long as we only use it for `Debug` builds and once we had different build configurations we could re-structure our `Fastfile`. What do you think of leaving it for now?